controllers/configurationpolicy_controller.go

```diff
@@ -1498,25 +1474,25 @@ func (r *ConfigurationPolicyReconciler) handleObjects(
 			"name", objName,
 			"kind", kindWithoutNS,
 		)
+
+		var space string
+		if objName != "" {
+			space = " "
+		}
+
 		// namespaced but none specified, generate violation
-		msg := fmt.Sprintf("namespaced object %s of kind %s has no namespace specified "+
+		msg := fmt.Sprintf("namespaced object%s%s of kind %s has no namespace specified "+
```

Comment:
When object name isn't specified, are you gonna add this error message..?  is there code for "object name isn't specified"?

Reply:
line 1500 in configuration_controller.go, filtering this case.